assets/react/controllers/Incident/List.jsx

```diff
@@ -0,0 +1,231 @@
+import React, { useEffect, useState, useMemo } from "react";
+import { Table, Tag, Avatar, Row, Col, Badge, Switch } from "antd";
+import TaskContext from "./TaskContext";
+import _ from "lodash";
+import "antd/dist/antd.css";
```

Comment:
This breaks the whole platform CSS, the fonts are overridden by ant.design. You shouldn't need to require it, or you should require the individual CSS files for Ant components, like in other JS code. 

<img width="1512" alt="Capture d’écran 2024-04-23 à 17 26 27" src="https://github.com/coopcycle/coopcycle-web/assets/1162230/9d7e719c-4668-4b44-a08d-d5d3e6fbe08b">